conf/machine/freedom-u540.conf

```diff
@@ -38,7 +38,7 @@ WIC_CREATE_EXTRA_ARGS ?= "--no-fstab-update"
 
 IMAGE_BOOT_FILES = ""
 
-TCLIBC = "glibc"
+TCLIBC ?= "glibc"
```

Comment:
musl does not work for riscv64 yet. Thats why it is locked to glibc. What is the usecase here ?

Reply:
Not allowing an override is confusing as if people set it they expect the override to work not just be ignored. If that causes a failure then so be it, if it silently works that is just confusing. There is a [musl port](https://github.com/riscv/riscv-musl) so maybe people want to use it.

Reply:
Yes, thats fine.putting TCLIBC here was done for ease of use and automation. I can change my workflow. Please remove it completely from here, since its a DISTRO property anyway.

Reply:
Removed